generator/lib/src/realm_model_info.dart

```diff
@@ -65,10 +65,7 @@ class RealmModelInfo {
         }
 
         yield* allExceptCollections.map((f) {
-          if (f.isFinal) {
-            return "RealmObject.set(this, '${f.name}', ${f.name});"; // since no setter will be created!
-          }
-          return 'this.${f.name} = ${f.name};'; // defer to generated setter
+          return "RealmObject.set(this, '${f.name}', ${f.name});";
```

Comment:
Isn't it better to be like this for all the properties with setters `return 'this.${f.name} = ${f.name};'; // defer to generated setter`?

Reply:
This change simplifies the code and is somewhat more performant. Why do you think it is better?

Reply:
Just suggesting because the code will be more readable. If the other way is more performant, OK.